Add link to Chrome CT Policy (#44630)
mdn/content: files/en-us/web/security/defenses/certificate_transparency/index.md

## Browser Requirements

> Chrome validates a website's TLS certificates for compliance against the [Chrome CT Policy](https://googlechrome.github.io/CertificateTransparency/ct_policy.html). Publicly-trusted certificates must be accompanied by SCTs satisfying the policy and successfully validate."

Reviewer: ```suggestion
Chrome validates a website's TLS certificates for compliance against the [Chrome CT Policy](https://googlechrome.github.io/CertificateTransparency/ct_policy.html). Publicly-trusted certificates must be accompanied by SCTs satisfying the policy and successfully validate.
```

> When a website's TLS certificate is validated in modern versions of Chrome, it is evaluated for compliance against the [Chrome CT Policy](https://googlechrome.github.io/CertificateTransparency/ct_policy.html). All publicly-trusted TLS certificates are required to be accompanied by SCTs that satisfy that policy to successfully validate.

Reviewer: Thanks @mathewhodson. Works for me, though I've removed the term "modern" - it ages worse than specific versions.

```suggestion
Chrome validates a website's TLS certificates for compliance against the [Chrome CT Policy](https://googlechrome.github.io/CertificateTransparency/ct_policy.html). Publicly-trusted certificates must be accompanied by SCTs satisfying the policy and successfully validate."
```
